docs/reinstallation/README.md

```diff
@@ -0,0 +1,383 @@
+3. [Enabling cluster reinstallation in the SiteConfig operator](#1-enabling-cluster-reinstallation-in-the-siteconfig-operator)  
+   - [Checking the current configuration](#checking-the-current-configuration)  
+   - [Enabling cluster reinstallation](#enabling-cluster-reinstallation)  
+4. [Labeling resources for preservation](#2-labeling-resources-for-preservation)  
+5. [Initiating a cluster reinstallation](#3-initiating-a-cluster-reinstallation)  
+   - [Updating the ClusterInstance resource](#updating-the-clusterinstance-resource)  
+   - [Applying the changes](#applying-the-changes)  
+6. [Optional. Monitoring the cluster reinstallation](#4-optional-monitoring-the-cluster-reinstallation)
+7. [Verify cluster availability](#5-verify-cluster-availability)
+8. [Advanced topics](#advanced-topics)  
+   - [Image-Based Break/Fix](#image-based-breakfix-for-single-node-openshift-hardware-replacement)
+
+</details>
+
+## Overview
+The SiteConfig operator simplifies OpenShift cluster reinstallation through the `ClusterInstance` API while preserving critical configuration data.
+
+With a GitOps-compatible, declarative approach, users can trigger reinstallations by updating the `ClusterInstance` resource. The operator also includes a backup and restore mechanism for `Secret` and `ConfigMap` resources, ensuring essential cluster data, such as authentication credentials and configuration resources, remains intact.
+
+### Cluster identity preservation
+Cluster reinstallation supports both single-node OpenShift and multi-node OpenShift clusters. However, cluster identity preservation is only supported for single-node OpenShift clusters installed using the Image Based Install provisioning method.
+
+## Cluster reinstallation workflow  
+To perform a cluster reinstallation, complete the following steps:
+
+1. [Enable the SiteConfig operator reinstallation service.](#1-enabling-cluster-reinstallation-in-the-siteconfig-operator)
+2. [Label resources for preservation.](#2-labeling-resources-for-preservation)
+3. [Initiate a cluster reinstallation.](#3-initiating-a-cluster-reinstallation)
+4. [Optional. Monitor the reinstallation progress.](#4-optional-monitoring-the-cluster-reinstallation)
+5. [Verify that the cluster is provisioned and available.](#5-verify-cluster-availability)
+
+You can complete step 1. and 2. well before of initiating the cluster reinstallation.
+
+### 1. Enabling cluster reinstallation in the SiteConfig operator
+
+You must explicitly enable cluster reinstallation in the `siteconfig-operator-configuration` `ConfigMap` resource, which you can complete well before initiating the process.
+
+#### Checking the current configuration  
+By default, reinstallation is disabled. You can check the current configuration by completing the following steps:
+
+Create the `NAMESPACE` environment variable for the `siteconfig-operator-configuration` `ConfigMap` resource. Run the following command:
+
+```sh
+NAMESPACE=<namespace>
+```
```

Comment:
Why do I need to define an env var for this process?

Reply:
Defining the `NAMESPACE` variable ensures clarity, reduces manual errors and decouples the namespace value (which can vary between `rhacm`, `open-cluster-management`, or `siteconfig-operator` depending on how the operator was installed) making the instructions flexible.